code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -740,3 +765,9 @@
 	required_reagents = list(/datum/reagent/acetone = 1, /datum/reagent/toxin/formaldehyde = 1, /datum/reagent/water = 1)
 	required_temp = 450
 
+/datum/chemical_reaction/holywater
+	name = "Holy Water"
+	id = /datum/reagent/water/holywater
+	results = list(/datum/reagent/water/holywater = 1)
+	required_reagents = list(/datum/reagent/water = 1)
```

Comment:
I think this was supposed to be datum/reagent/water/hollowwater. Unless you mean for holy water to be mass-producible without geysers, which I'd be fine with.

Reply:
Woops, it was very late. Thanks, I’ll double check the rest aswell